catalyst-gateway/bin/src/db/index/queries/staked_ada/get_txo_by_stake_address.rs

```diff
@@ -1,24 +1,37 @@
 //! Get the TXO by Stake Address
-use std::sync::Arc;
+use std::sync::{Arc, LazyLock};
 
 use cardano_blockchain_types::StakeAddress;
-use scylla::{
-    prepared_statement::PreparedStatement, transport::iterator::TypedRowStream, DeserializeRow,
-    SerializeRow, Session,
-};
+use futures::TryStreamExt;
+use moka::{ops::compute::Op, policy::EvictionPolicy};
+use scylla::{prepared_statement::PreparedStatement, DeserializeRow, SerializeRow, Session};
 use tracing::error;
 
-use crate::db::{
-    index::{
-        queries::{PreparedQueries, PreparedSelectQuery},
-        session::CassandraSession,
+use super::update_txo_spent::UpdateTxoSpentQueryParams;
+use crate::{
+    db::{
+        index::{
+            queries::{PreparedQueries, PreparedSelectQuery},
+            session::CassandraSession,
+        },
+        types::{DbSlot, DbStakeAddress, DbTransactionId, DbTxnIndex, DbTxnOutputOffset},
     },
-    types::{DbSlot, DbStakeAddress, DbTransactionId, DbTxnIndex, DbTxnOutputOffset},
+    settings::Settings,
 };
 
 /// Get txo by stake address query string.
 const GET_TXO_BY_STAKE_ADDRESS_QUERY: &str = include_str!("../cql/get_txo_by_stake_address.cql");
 
+/// In memory cache of the most recent Cardano UTXO assets by Stake Address.
+static ASSETS_CACHE: LazyLock<moka::future::Cache<DbStakeAddress, Vec<GetTxoByStakeAddressQuery>>> =
```

Comment:
```suggestion
static ASSETS_CACHE: LazyLock<moka::future::Cache<DbStakeAddress, Aec<Vec<GetTxoByStakeAddressQuery>>>> =
```

See: https://github.com/moka-rs/moka/tree/main?tab=readme-ov-file#avoiding-to-clone-the-value-at-get